mod-source-record-storage-server/src/main/java/org/folio/services/handlers/match/AbstractMarcMatchEventHandler.java

```diff
@@ -137,8 +138,9 @@ private Future<List<Record>> retrieveMarcRecords(MatchField matchField, String t
       LOG.debug("retrieveMarcRecords:: Process default field matching, matchField {}, tenant {}", matchField, tenant);
       return processDefaultMatchField(matchField, tenant).map(RecordCollection::getRecords);
     }
+    //TODO is Filter.ComparisonPartType needed?
```

Comment:
looks like it is needed to support ComparisonPartType option for marc-authority to marc-authority records matching

Reply:
completed, thanks

Reply:
thanks